apps/web/src/components/landing/Footer.tsx

```diff
@@ -1,23 +1,170 @@
 "use client";
 
-import { type FunctionComponent, useState } from "react";
+import { useEffect, useState } from "react";
 import Image from "next/image";
 import Link from "next/link";
-import { Instagram, Facebook, Twitter, Github, Disc } from "lucide-react";
-import Discord from "../../../public/img/landing/discord_icon.svg";
+import { Instagram, Facebook, Twitter, Github } from "lucide-react";
+import {
+  DropdownMenu,
+  DropdownMenuTrigger,
+  DropdownMenuItem,
+  DropdownMenuContent,
+} from "../shadcn/ui/dropdown-menu";
 
 interface Props {
-	className?: string;
+  className?: string;
 }
 
+interface LinkData {
+  name: string;
+  link: string;
+}
+
+function linkItem(name: string, link: string): LinkData {
+  return { name, link };
+}
+
+// Maybe source this from the configuration file so that other hackathons can define their links in a more declarative way.
+const dropdownItemMap = {
```

Comment:
Agreed. Can this also be moved to the config if it is not too much trouble?

Reply:
should be implemented by 8048739